dogs.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Nice!

Also, DEATH to internet explorer! Which is why we need this tag. =)

Reply:
LOL!

Reply:
The project I'm working on is going to be used by mostly IE11 users and I hate it